packages/strapi-admin/admin/src/containers/LocaleToggle/styles.scss

```diff
@@ -61,7 +61,7 @@
   left: auto !important;
   right: -5px !important;
   min-width: 90px!important;
-  max-height: 162px;
+  max-height: auto !important;
```

Comment:
I would prefer to not set the `max-height` to `auto`. Can you try to add `overflow: visible` instead? 

This languages list was thinking as a scrollable list 👍

Reply:
In case `overflow: visible', language list are overflow.`overflow: auto` would be better. 

![image](https://user-images.githubusercontent.com/13197080/42417241-28c920e8-82c0-11e8-959d-2528eee8fdc5.png)